packages/wallet-backend-update/src/account/service.ts

```diff
@@ -198,6 +201,50 @@ export class AccountService implements IAccountService {
     }
   }
 
+  public async withdrawFunds(args: WithdrawFundsArgs): Promise<void> {
+    const account = await this.findAccountById(args.accountId, args.userId)
+
+    // get list of payout method types for currency, if we get to production: get payout type required fields will be needed
+    const payoutType = await this.deps.rapyd.getPayoutMethodTypes(
```

Comment:
I would argue that payoutType can also be private inside rapyd-client, and used by the withdrawFundsAccount, since payoutType also has no other use here in the accountService. (or does it?)

(Ditto to the completePayout made as private suggestion)